funcoeszz

```diff
@@ -94,12 +93,32 @@ unset ZZCOR_DFT ZZPATH_DFT ZZDIR_DFT ZZTMPDIR_DFT
 while test $# -gt 0
 do
 	case "$1" in
-		--path) ZZPATH="$2"   ; shift; shift ;;
-		--dir ) ZZDIR="${2%/}"; shift; shift ;;
-		--cor ) ZZCOR="$2"    ; shift; shift ;;
+		--path   ) ZZPATH="$2"   ; shift; shift ;;
+		--dir    ) ZZDIR="${2%/}"; shift; shift ;;
+		--cor    ) ZZCOR="$2"    ; shift; shift ;;
+		--browser) ZZBROWSER="$2"; shift; shift ;;
 		*) break;;
 	esac
 done
+
+#
+### Configura os comandos para baixar arquivos
+#
+case "$ZZBROWSER" in
+	links)
+		ZZWWWDUMP='links -dump'
+		ZZWWWLIST='?fixme?'
```

Comment:
Show! Tou na rua agora, fica a vontade pra mandar commits pra essa branch,
até deixarmos isso redondo.

Podia entrar um elinks, wget, curl também. Apesar desses dois não
interpretarem HTML :/
Em 24/03/2016 19:15, "itamarnet" <notifications@github.com> escreveu:

> In funcoeszz
> <https://github.com/funcoeszz/funcoeszz/pull/241#discussion_r57396557>:
>
> >  		*) break;;
> >  	esac
> >  done
> > +
> > +#
> > +### Configura os comandos para baixar arquivos
> > +#
> > +case "$ZZBROWSER" in
> > +	links)
> > +		ZZWWWDUMP='links -dump'
> > +		ZZWWWLIST='?fixme?'
>
> Para esse caso pode ser:
>
> ZZWWWLIST='links -html-numbered-links 1'
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/funcoeszz/funcoeszz/pull/241/files/8753ef48b984cbf0d8a6ea8f186f690bfc80385c#r57396557>
>